docs/docsite/rst/reference_appendices/glossary.rst

```diff
@@ -354,12 +354,11 @@ when a term comes up on the mailing list.
         Ansible orchestration is all about modeling that kind of process.
 
     paramiko
-        By default, Ansible manages machines over SSH.   The library that
-        Ansible uses by default to do this is a Python-powered library called
-        paramiko.  The paramiko library is generally fast and easy to manage,
-        though users who want to use Kerberos or Jump Hosts may wish to switch
-        to a native SSH binary such as OpenSSH by specifying the connection
-        type in their :term:`playbooks`, or using the ``-c ssh`` flag.
+        By default, Ansible uses native OpenSSH, because it supports ControlPersist.
+        If your control machine uses an older version of OpenSSH that does not support ControlPersist,
```

Comment:
Drop the two lines about ControlPersist. It's superfluous information for the glossary entry. It also duplicates the details from here: https://docs.ansible.com/ansible/latest/inventory_guide/connection_details.html#controlpersist-and-paramiko

If that changes, you'd need to update that section as well as the glossary.